nss-day-cohort-34/workforce-management-awkward-conversations-and-gravy
# Computers

# Description

User can view all computers, view one computer, create a computer, and delete a computer if it has never been assigned to an employee.

Related to Issue # 7


# Testing Instructions
Create a computer
View Details
Try to delete a computer with a Decommission Date listed (You should see a statement telling you it can't be deleted instead of a delete button)
Try to delete a computer without a Decommission Date listed (You should see the delete button and be able to click it). The computer should then be deleted.



# Checklist:
